patches/tModLoader/Terraria/GameContent/UI/ResourceSets/ResourceDrawSettings.cs.patch

```diff
@@ -0,0 +1,61 @@
+--- src/TerrariaNetCore/Terraria/GameContent/UI/ResourceSets/ResourceDrawSettings.cs
++++ src/tModLoader/Terraria/GameContent/UI/ResourceSets/ResourceDrawSettings.cs
+@@ -1,6 +_,7 @@
+ using Microsoft.Xna.Framework;
+ using Microsoft.Xna.Framework.Graphics;
+ using ReLogic.Content;
++using Terraria.ModLoader;
+ 
+ namespace Terraria.GameContent.UI.ResourceSets
+ {
+@@ -16,6 +_,9 @@
+ 		public Vector2 OffsetPerDrawByTexturePercentile;
+ 		public Vector2 OffsetSpriteAnchor;
+ 		public Vector2 OffsetSpriteAnchorByTexturePercentile;
++		public PlayerStatsSnapshot StatsSnapshot;
++		public IPlayerResourcesDisplaySet DisplaySet;
++		public int ResourceSetSlot;
```

Comment:
`internal`?

Reply:
ResourceDrawSettings is public, so if those fields were made internal, modders would be blocked from using the struct properly without resorting to Reflection